SimPEG/Maps.py

```diff
@@ -389,10 +389,10 @@ def nP(self):
 class SelfConsistentEffectiveMedium(IdentityMap, properties.HasProperties):
     """
         Two phase self-consistent effective medium theory mapping for
-        ellipsoidal inclusions. The model is the concentration
+        ellipsoidal inclusions. The inversion model is the concentration
```

Comment:
@sgkang: I specified that the inversion model is concentration here

Reply:
Thanks @lheagy !